website/docs/r/service_dictionary_items_v1.html.markdown

```diff
@@ -0,0 +1,212 @@
+      comment = "demo"
+  }
+
+  backend {
+      address = "demo.notexample.com.s3-website-us-west-2.amazonaws.com"
+      name    = "AWS S3 hosting"
+      port    = 80
+    }
+
+  dictionary {
+	name       = var.mydict_name
+  }
+
+  force_destroy = true
+}
+
+resource "fastly_service_dictionary_items_v1" "items" {
+    service_id = "${fastly_service_v1.myservice.id}"
+    dictionary_id = "${{for s in fastly_service_v1.myservice.dictionary : s.name => s.dictionary_id}[var.mydict_name]}"
+    items = {
+        key1: "value1"
+        key2: "value2"
+    }
+}
+
+```
+
+Complex object usage:
+
+```hcl
+
+variable "mydict" {
+  type = object({ name=string, items=map(string) })
+  default = {
+    name = "My Dictionary"
+    items = {
+      key1: "value1x"
+      key2: "value2x"
+    }
+  }
+}
+
+resource "fastly_service_v1" "myservice" {
+  name = "demofastly"
+
+  domain {
+    name    = "demo.notexample.com"
+    comment = "demo"
+  }
+
+  backend {
+    address = "demo.notexample.com.s3-website-us-west-2.amazonaws.com"
+    name    = "AWS S3 hosting"
+    port    = 80
+  }
+
+  dictionary {
+    name = var.mydict.name
+  }
+
+  force_destroy = true
+  }
+
+resource "fastly_service_dictionary_items_v1" "items" {
+  service_id = fastly_service_v1.myservice.id
+  dictionary_id = {for d in fastly_service_v1.myservice.dictionary : d.name => d.dictionary_id}[var.mydict.name]
+  items = var.mydict.items
+}
+
+```
+
+Expression and functions usage:
+
+```hcl
+// Local variables used when formating values for the "My Project Dictionary" example
+locals {
+  dictionary_name = "My Project Dictionary"
+  host_base = "demo.ocnotexample.com"
+  host_divisions = ["alpha", "beta", "gamma", "delta"]
+}
+
+// Define the standard service that will be used to manage the dictionaries.
+resource "fastly_service_v1" "myservice" {
+  name = "demofastly"
+
+  domain {
+    name = "demo.ocnotexample.com"
+    comment = "demo"
+  }
+
+  backend {
+    address = "demo.ocnotexample.com.s3-website-us-west-2.amazonaws.com"
+    name = "AWS S3 hosting"
+    port = 80
+  }
+
+  dictionary {
+    name = local.dictionary_name
+  }
+
+  force_destroy = true
+}
+
+// This resource is dynamically creating the items from the local variables through for expressions and functions.
+resource "fastly_service_dictionary_items_v1" "project" {
+  service_id = fastly_service_v1.myservice.id
+  dictionary_id = {for d in fastly_service_v1.myservice.dictionary : d.name => d.dictionary_id}[local.dictionary_name]
+  items = {
+    for division in local.host_divisions:
+      division => format("%s.%s", division, local.host_base)
+  }
+
+}
+```
+
+Lifecyle ignore_changes usage:
+
+The following example demonstrates how the lifecycle ignore_change field can be used to suppress updates against the 
+items in a dictionary.  If an external means, (for example via the Fastly API or UI) is used to manage items in a dictionary, then this will stop Terraform realigning the remote state.
```

Comment:
```suggestion
items in a dictionary.  If, after your first deploy, the Fastly API or UI is to be used to manage items in a dictionary, then this will stop Terraform realigning the remote state with the initial set of dictionary items defined in your HCL.
```